Jenkinsfile

```diff
@@ -29,7 +38,7 @@ pipeline {
         }
         stage('Cleanup') {
             steps {
-                sh "rm -rfv build"
+                sh "rm -rfv build endless-key-config"
```

Comment:
Since the same repo is always used, it would be a little friendlier not to delete it so that a fresh clone isn't needed every time. It doesn't hurt, though.